pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1594,6 +1602,25 @@ def _check_use_list_or_dict_literal(self, node: nodes.Call) -> None:
                 elif inferred.qname() == "builtins.dict" and not node.keywords:
                     self.add_message("use-dict-literal", node=node)
 
+    def _check_use_dict_literal(self, node: nodes.Call) -> None:
+        """Check if dict is created by using the literal {}."""
+        if not isinstance(node.func, astroid.Name):
+            return
+        inferred = utils.safe_infer(node.func)
+        if (
+            isinstance(inferred, nodes.ClassDef)
+            and inferred.qname() == "builtins.dict"
+            and not node.args
+            and node.keywords
+        ):
+            code_suggestion = next(node.infer()).as_string()
+            short_suggestion = (
+                f"{code_suggestion[:100]}{'...' if len(code_suggestion) > 100 else ''}"
+            )
```

Comment:
I'm limiting the dict output to 100 characters here. What do you think, @Pierre-Sassoulas?

I couldn't find any other examples in the code.